pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -40,102 +41,102 @@ func GetAllFSTypesExcept(exceptions ...string) sets.String {
 func GetAllFSTypesAsSet() sets.String {
 	fstypes := sets.NewString()
 	fstypes.Insert(
-		string(policy.HostPath),
-		string(policy.AzureFile),
-		string(policy.Flocker),
-		string(policy.FlexVolume),
-		string(policy.EmptyDir),
-		string(policy.GCEPersistentDisk),
-		string(policy.AWSElasticBlockStore),
-		string(policy.GitRepo),
-		string(policy.Secret),
-		string(policy.NFS),
-		string(policy.ISCSI),
-		string(policy.Glusterfs),
-		string(policy.PersistentVolumeClaim),
-		string(policy.RBD),
-		string(policy.Cinder),
-		string(policy.CephFS),
-		string(policy.DownwardAPI),
-		string(policy.FC),
-		string(policy.ConfigMap),
-		string(policy.VsphereVolume),
-		string(policy.Quobyte),
-		string(policy.AzureDisk),
-		string(policy.PhotonPersistentDisk),
-		string(policy.StorageOS),
-		string(policy.Projected),
-		string(policy.PortworxVolume),
-		string(policy.ScaleIO),
-		string(policy.CSI),
+		string(policyv1beta1.HostPath),
+		string(policyv1beta1.AzureFile),
+		string(policyv1beta1.Flocker),
+		string(policyv1beta1.FlexVolume),
+		string(policyv1beta1.EmptyDir),
+		string(policyv1beta1.GCEPersistentDisk),
+		string(policyv1beta1.AWSElasticBlockStore),
+		string(policyv1beta1.GitRepo),
+		string(policyv1beta1.Secret),
+		string(policyv1beta1.NFS),
+		string(policyv1beta1.ISCSI),
+		string(policyv1beta1.Glusterfs),
+		string(policyv1beta1.PersistentVolumeClaim),
+		string(policyv1beta1.RBD),
+		string(policyv1beta1.Cinder),
+		string(policyv1beta1.CephFS),
+		string(policyv1beta1.DownwardAPI),
+		string(policyv1beta1.FC),
+		string(policyv1beta1.ConfigMap),
+		//string(policyv1beta1.VsphereVolume),
```

Comment:
I understand that you commented it out because the const didn't exist in v1beta1. I think we have to add them to v1beta1 first, then merge this PR.

Reply:
its done by #69704 #69795